swagger-spec/licenses/list.yaml

```diff
@@ -1 +1,51 @@
 # /licenses/
+get:
+  summary: List all licenses
+  description: >-
+    A paginated list of licenses.
+    The returned preprints are sorted by their creation date, with the most recent
+    preprints appearing first.
+
+    #### Returns
+
+    A dictionary with a `data` and `links` property.
+
+
+    The `data` property contains an array of 10 licenses.
+    Each resource in the array is a separate license object.
+    The `links` property contains a dictionary of links that can be used
+    for [pagination](http://jsonapi.org/format/1.0/#fetching-pagination).
+
+
+    This request should never return an error.
+
+    #### Filtering
+
+    You can optionally request that the response only include licenses that
+    match your filters by utilizing the `filter` query parameter, e.g.
+    https://api.osf.io/v2/licenses/?filter[name]=open.
```

Comment:
This filter returns 0 results, so let's use https://api.osf.io/v2/licenses/?filter[name]=apache instead.